scripts/postgresql_install_binaries.sh

```diff
@@ -17,6 +17,12 @@ sudo yum-config-manager --enable ol7_developer_EPEL
 sudo yum install -y llvm5.0-devel
 sudo yum install -y postgresql${pg_version_no_dot}-devel
 
+# Setting firewall rules
+sudo -u root bash -c "firewall-cmd --permanent --zone=trusted --add-source=${pg_whitelisted_ip}/32"
```

Comment:
Use the conditional execution as follows:
```
export pg_whitelisted_ip='${pg_whitelisted_ip}'
if [[ $pg_whitelisted_ip != "" ]]; then 
   sudo -u root bash -c "firewall-cmd --permanent --zone=trusted --add-source=${pg_whitelisted_ip}/32"
fi
```
The idea behind it is to skip firewall-cmd execution when the `pg_whitelisted_ip` variable has an empty value. It will prevent the error.